src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -1331,6 +1364,10 @@
 helps['network nsg'] = """
     type: group
     short-summary: Manage Azure Network Security Groups (NSGs).
+    long-summary: You can control network traffic to resources in a virtual network using a network security
```

Comment:
There is a disparity in many of these `long-summary` elements: Some only link out to the external site, and some include lots of additional information (like this one.) I would recommend the following policy, which may already be followed in this pull request:

* `group` help should include a longer description like this one, as well as a link.
* `command` help should include only the link, __unless__ there are relevant restrictions or other factors which must be documented.

Reply:
Also it should be noted some links are `aka.ms` and some links are a full docsite URL. It would be useful to standardize on one format, probably shortened URLs.